src/indigo/less/head.less

```diff
@@ -39,6 +39,7 @@
         }
     }
 
+    > a,
```

Comment:
asi by som to este doladil ... kludne daj `.kbc-notification-access` separatne. Pretoze teraz to navadza, ze niekde  existuje `.kbc-notification-access > .kbc-icon-keboola`, co podla mna nie je pravda.